configure

```diff
@@ -86,23 +86,23 @@ EOF
 
 found=no
 for lib in "-lc" "-lfts"; do
-  # shellcheck disable=SC2086
-  ${CC} $CFLAGS $LDFLAGS conftest.c -Wl,$lib >/dev/null 2>&1
-	ret=$?
-	if test $ret -eq 0; then
-		FTS_LIBS="$lib"
-		found=yes
-		break;
-	fi
+    # shellcheck disable=SC2086
+    ${CC} $CFLAGS $LDFLAGS conftest.c -Wl,$lib > /dev/null 2>&1
+    ret=$?
+    if test $ret -eq 0; then
+        FTS_LIBS="$lib"
+        found=yes
+        break
```

Comment:
Is the removal of the semicolon here ```break``` from the previous statement ```break;``` intentional or an oversight?

Reply:
a semicolon at the end of a line is redundant in shell